### DISCUSSION TITLE
Retrowave race map.

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1210] [Maps and Content Showcase]
A race map that is set in a very abstract, retrofuturistic/virtual-like theme, and uses a lot of md3 models to give it life, along with a lot of moving platforms and obstacles.
There is a bunch of leaps of faith, so be careful!

![Hnet-image (3)](https://user-images.githubusercontent.com/24359960/93719301-efac6280-fb57-11ea-8e8d-05bd3c88b3f6.gif)

![20200920150202](https://user-images.githubusercontent.com/24359960/93719123-fdadb380-fb56-11ea-958f-e0e05dd52799.png)
![20200920150218](https://user-images.githubusercontent.com/24359960/93719124-fe464a00-fb56-11ea-81b3-a3b2d82d00aa.png)
![20200920150229](https://user-images.githubusercontent.com/24359960/93719125-fe464a00-fb56-11ea-8915-6817b5ee9d89.png)
![20200920150246](https://user-images.githubusercontent.com/24359960/93719126-fedee080-fb56-11ea-85e4-9bfa08bdb236.png)
![Hnet com-image (1)](https://user-images.githubusercontent.com/24359960/93719336-4a45be80-fb58-11ea-9cca-ec0437b7bfb2.gif)

Alternate variant has no fog, and a brighter color pallete.
![Hnet-image (6)](https://user-images.githubusercontent.com/24359960/93719607-0eabf400-fb5a-11ea-8fa3-cf28d8926769.gif)

[retrowave.zip](https://github.com/redeclipse/world/files/5252403/retrowave.zip)

PS: This is RE 2.1 only, as it uses the rail entity system for the moving obstacles.

**Comment by Ulukai55** [comment]
Well look at that! Crazy effects! Definitely going to try this one. I like the atmosphere you created. Nice work.

**Comment by qreeves** [comment]
Needs a good music track to go with it lol.

**Comment by SniperGoth** [comment]
Update (V2):
-Gameplay
Reworked some obstacles, so the map is a bit more interesting, and easier to play through.

![ezgif-5-ac6e3e2431](https://github.com/redeclipse/base/assets/24359960/6a256f1d-189e-4cfe-bbaf-8d67867c50b5)
![ezgif-5-b39bcfe089](https://github.com/redeclipse/base/assets/24359960/c3ea1cd9-3e0a-4db5-b225-afbec1348f4d)


-Visual changes
Added weather effects, clouds with the grid texture, virtual "mountains" in the background, along with various other things, like volumetric fog, sound envs etc.
![20240104_164703](https://github.com/redeclipse/base/assets/24359960/721a4506-b12a-48a0-be72-c294dd0a51cd)
![20240104_163400](https://github.com/redeclipse/base/assets/24359960/0e8ff043-aae3-438c-b27c-215a4e2cd708)
![20240104_164602](https://github.com/redeclipse/base/assets/24359960/40a75d12-22e7-4716-9621-0b611f658645)
![20240104_170256](https://github.com/redeclipse/base/assets/24359960/6b7819c4-5db0-4a94-a00d-9b0c4ba5ac0c)

Displacement textures:
![ezgif-5-55620baded](https://github.com/redeclipse/base/assets/24359960/3c757937-f111-404e-97b6-e24bda33277c)

Time to beat, if you use the shortcuts is roughly 1:40 minutes.
![20240104_163224](https://github.com/redeclipse/base/assets/24359960/fc46f9ae-5529-4fbd-9fbf-09341f54576c)

There is some minor polishing and ideas i want to add, still (not any related to gameplay), but i just want to see if it is a improvement over the original at this point.
It's been too long.

Map file (only new thing included here is the distance mountain image, which i made on blender):
[retrowavev2.zip](https://github.com/redeclipse/base/files/13835150/retrowavev2.zip)
